controller/src/main/java/io/pravega/controller/server/rpc/auth/PasswordAuthHandler.java

```diff
@@ -71,23 +74,25 @@ public String getHandlerName() {
     }
 
     @Override
-    public boolean authenticate(String token) {
+    public Principal authenticate(String token) throws AuthException {
         String[] parts = parseToken(token);
         String userName = parts[0];
         String password = parts[1];
 
         try {
-            return userMap.containsKey(userName) && encryptor.checkPassword(password, userMap.get(userName).encryptedPassword);
+            if (userMap.containsKey(userName) && encryptor.checkPassword(password, userMap.get(userName).encryptedPassword)) {
+                return new UnixPrincipal(userName);
```

Comment:
Can't use `UnixPrincipal` because `com.sun.*` has issues in JDK9.  

I suggest simply introducing a `DefaultPrincipal` that simply carries the username.

Reply:
Can you point a reference to the issues? `UnixPrincipal` seems too entrenched to be deprecated!

Reply:
This is related to JDK9 modularization ("jigsaw"). You're right that I'm being overly conservative in the case of `UnixPrincipal`; the `jdk.security.auth` module seems to export it for normal use.  
https://docs.oracle.com/javase/9/docs/api/jdk.security.auth-summary.html

Still feels wrong to me, but not a big deal.

Reply:
Please create an issue for this if you don't want to resolve here.

Reply:
Here is an issue to track this: https://github.com/pravega/pravega/issues/2821